libraries/psibase/common/include/psibase/blob.hpp

```diff
@@ -4,16 +4,29 @@
 #include <algorithm>
 #include <type_traits>
 
+template <typename T>
+concept IsByteType =
+    std::is_same_v<std::decay_t<T>, char> || std::is_same_v<std::decay_t<T>, unsigned char>;
```

Comment:
`std::decay_t` should be applied outside this concept. I don't think that `IsByteType<char&>` should be true. Also, `remove_cvref` is usually preferable to `decay`.

Reply:
done/done